music21/romanText/writeRoman.py

```diff
@@ -0,0 +1,552 @@
+    Here's the last in our example:
+
+    >>> rnWriterFromScore.combinedList[-1]
+    'm10 b1 V6/V b2 V b3 I'
+
+    In the case of the score, the top part is assumed to contain the Roman numerals.
+    This is consistent with the parsing of rntxt which involves putting Roman numerals in a part
+    (the top, and only part) within a score.
+
+    In all other cases, the objects are iteratively inserted into larger streams until we end up
+    with a part object (e.g. measure > part).
+
+    >>> rn = roman.RomanNumeral('viio64', 'a')
+    >>> rnWriterFromRn = romanText.writeRoman.RnWriter(rn)
+    >>> rnWriterFromRn.combinedList[0]
+    'Composer: Composer unknown'
+
+    >>> rnWriterFromRn.combinedList[-1]
+    'm0 b1 a: viio64'
+
+    OMIT_FROM_DOCS
+
+    Users can do these insertions themselves, but don't need to:
+
+    >>> m = stream.Measure()
+    >>> m.insert(0, rn)
+    >>> rnWriterFromMeasure = romanText.writeRoman.RnWriter(m)
+    >>> rnWriterFromMeasure.combinedList == rnWriterFromRn.combinedList
+    True
+
+    >>> p = stream.Part()
+    >>> p.insert(0, m)
+    >>> rnWriterFromPart = romanText.writeRoman.RnWriter(p)
+    >>> rnWriterFromPart.combinedList == rnWriterFromMeasure.combinedList
+    True
+
+    >>> s = stream.Score()
+    >>> s.insert(0, m)
+    >>> rnWriterFromScoreWithoutPart = romanText.writeRoman.RnWriter(p)
+    >>> rnWriterFromScoreWithoutPart.combinedList == rnWriterFromMeasure.combinedList
+    True
+    '''
+
+    def __init__(self,
+                 obj: base.Music21Object,
+                 ):
+
+        self.composer = 'Composer unknown'
+        self.title = 'Title unknown'
+
+        if obj.isStream:
+
+            if isinstance(obj, stream.Opus):
+                constituentElements = [RnWriter(x) for x in obj]
+                self.combinedList = []
+                for scoreOrSim in constituentElements:
+                    for x in scoreOrSim.combinedList:
+                        self.combinedList.append(x)
+                    self.combinedList.append('\n')  # one between scores
+                return
+
+            elif isinstance(obj, stream.Score):
+                if obj.parts:
+                    self.container = obj.parts[0]
+                else:  # score with no parts
+                    self.container = obj
+
+            elif isinstance(obj, stream.Part):
+                self.container = obj
+
+            elif isinstance(obj, stream.Measure):
+                self.container = stream.Part()
+                self.container.insert(0, obj)
+
+            else:  # A stream, but not a measure, part, or score
+                self._makeContainer(obj)
+
+            if obj.metadata:  # Check the obj (not container) for metadata if obj is a stream
+                self.prepTitle(obj.metadata)
+                if obj.metadata.composer:
+                    self.composer = obj.metadata.composer
+
+        else:  # Not a stream
+            self._makeContainer([obj])
```

Comment:
WOW!  The code coverage of this module is SPECTACULAR!  This is actually the only uncovered line -- not sure if you want to write a unittest (better not doctest) to cover it to have the (first?) long module in music21 that is 100% tested?  If not, no problem, but WOW!

Reply:
Thanks, and hurrah!

I'm happy to add another test to this effect but both the `RnWriter` docs and the `testTypeParses` unittest include calling this class directly on a RomanNumeral object (which is obviously not a stream) so it's not clear why this line isn't registering as covered.